saidump/saidump.cpp

```diff
@@ -31,11 +41,13 @@ void printUsage()
 {
     SWSS_LOG_ENTER();
 
-    std::cout << "Usage: saidump [-t] [-g] [-h]" << std::endl;
+    std::cout << "Usage: saidump [-t] [-g] [-r] [-h]" << std::endl;
     std::cout << "    -t --tempView:" << std::endl;
     std::cout << "        Dump temp view" << std::endl;
     std::cout << "    -g --dumpGraph:" << std::endl;
     std::cout << "        Dump current graph" << std::endl;
+    std::cout << "    -r --rdb:" << std::endl;
+    std::cout << "        Dump by parsing the Redis persistence file dump.rdb that is generated by SAVE command." << std::endl;
```

Comment:
no other command description has dot (.) at the end of description, please remove

Reply:
Fix it.